src/js/common/components/AlertBox.jsx

```diff
@@ -52,13 +51,33 @@ class AlertBox extends React.Component {
       );
     }
 
+    if (this.props.headline) {
+      return (
+        <div
+          aria-live="assertive"
+          className={alertClass}
+          ref={(ref) => { this._ref = ref; }}>
+          <div className="usa-alert-body">
+            <div className="usa-alert-heading">
+              {this.props.headline}
+            </div>
+            <div className="usa-alert-text">
+              {this.props.content}
+            </div>
+          </div>
+          {closeButton}
+          <div className="cf"></div>
+        </div>
+      );
+    }
+
     return (
       <div
         aria-live="assertive"
         className={alertClass}
         ref={(ref) => { this._ref = ref; }}>
-        <div className="va-alert-body usa-alert-body">
-          <div className="usa-alert-text usa-alert-heading">
+        <div className="usa-alert-body">
+          <div className="usa-alert-text">
             {this.props.content}
           </div>
         </div>
```

Comment:
There's a section in the standards.usa.gov docs for accessibility in the alert boxes that mentions `role="alert"` and some other stuff we may want to include

Reply:
I'm going to file a ticket for this -- we should do an accessibility pass across the board.